sphinx/templates/quickstart/conf.py_t

```diff
@@ -110,12 +114,15 @@ html_static_path = ['{{ dot }}static']
 # Custom sidebar templates, must be a dictionary that maps document names
 # to template names.
 #
-# This is required for the alabaster theme
-# refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
+# For alabaster, check
+# http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
 html_sidebars = {
     '**': [
-        'relations.html',  # needs 'show_related': True theme option to display
-        'searchbox.html',
+        'about.html',          # comment this out if not using alabaster theme
+        #  'navigation.html',  # specific to alabaster (and possibly others)
```

Comment:
perhaps this one should be uncommented-out to with instructions to comment itout if not using alabaster theme